modules_core/com.etendorx.auth/src/main/java/com/etendorx/auth/JwtauthApplication.java

```diff
@@ -6,8 +6,7 @@
 import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
-@ComponentScan(basePackages = { "com.etendorx.clientrest.*", "com.etendorx.auth.*",
-    "com.etendorx.utils.auth.key.context" })
+@ComponentScan(basePackages = { "com.etendorx.clientrest.*", "com.etendorx.auth.*", "com.etendorx.auth.auth"})
```

Comment:
Is it needed to scan "com.etendorx.auth.auth" when you already used "com.etendorx.auth.*" before?

Also, is it OK to use the "*"?